adapters/repos/db/lsmkv/compaction_integration_test.go

```diff
@@ -10,7 +10,6 @@
 //
 
 //go:build integrationTest
-// +build integrationTest
```

Comment:
Sort of unrelated to the PR, I don't know enough about our go test build pipeline to know that is the difference in this change, but I'm curious on what we define as a regular and an integration test and when they run.

Reply:
I think this was required in old go versions - my IDE tells me that this is not required anymore and I delete it when I see it. Nothing broke, so I assume it is fine 🤷